src/main/java/com/google/graphgeckos/dashboard/datatypes/BuildInfo.java

```diff
@@ -124,6 +135,24 @@ public void addBuilder(@NonNull BuildBotData update) {
     builders.add(update);
   }
 
+  private RevisionStatus analyseRevision() {
+    boolean allPassed = false;
```

Comment:
I don't follow: the code below would mark `allPassed` to `true` if the builder statuses are [LOST, PASSED, LOST] for example. Maybe you need a different name for this?

I would be helpful to add a javadoc describing the cases where the different statuses get matched.

Reply:
I think the variable name is misleading. The status should indeed be true in the case you described, since it's the only valid datapoint that we have. I'll change the name and add a javadoc to the method